packages/contracts-bedrock/deploy-config/goerli.json

```diff
@@ -1,26 +1,39 @@
 {
-  "l1StartingBlockTag": "0xafce66a0a2446856112e4069b275ad32b1f4a607888f9c4c59eddf9be81f8670",
+  "finalSystemOwner": "DUMMY",
+  "controller": "DUMMY",
+
+  "l1StartingBlockTag": "DUMMY",
   "l1ChainID": 5,
-  "l2ChainID": 111,
+  "l2ChainID": 420,
   "l2BlockTime": 2,
 
-  "maxSequencerDrift": 1000,
-  "sequencerWindowSize": 120,
+  "maxSequencerDrift": 3600,
+  "sequencerWindowSize": 7200,
   "channelTimeout": 120,
-  "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
-  "batchInboxAddress": "0xff00000000000000000000000000000000000002",
-  "batchSenderAddress": "0xa11d2b908470e17923fff184d48269bebbd9b2a5",
-
-  "systemConfigOwner": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
 
-  "l2OutputOracleSubmissionInterval": 6,
+  "p2pSequencerAddress": "DUMMY",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000000",
+  "batchSenderAddress": "DUMMY",
+  "l2OutputOracleSubmissionInterval": 20,
   "l2OutputOracleStartingTimestamp": -1,
-  "l2OutputOracleProposer": "0x6c23a0dcdfc44b7a57bed148de598895e398d984",
-  "l2OutputOracleChallenger": "0x6925b8704ff96dee942623d6fb5e946ef5884b63",
+  "l2OutputOracleProposer": "DUMMY",
+  "l2OutputOracleChallenger": "DUMMY",
+  "finalizationPeriodSeconds": 2,
+
+  "systemConfigOwner": "DUMMY",
+
+  "l2GenesisBlockGasLimit": "0x1C9C380",
+  "l2GenesisBlockCoinbase": "0x42000000000000000000000000000000000000f0",
+  "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
+
+  "optimismBaseFeeRecipient": "DUMMY",
+  "optimismL1FeeRecipient": "DUMMY",
 
-  "baseFeeVaultRecipient": "0xf116a24056b647e3211d095c667e951536cdebaa",
-  "l1FeeVaultRecipient": "0xc731837b696ca3d9720d23336925368ceaa58f83",
-  "sequencerFeeVaultRecipient": "0xf116a24056b647e3211d095c667e951536cdebaa",
+  "l2CrossDomainMessengerOwner": "DUMMY",
 
-  "deploymentWaitConfirmations": 1
+  "gasPriceOracleOverhead": 2100,
```

Comment:
In theory the overhead will be able to be much smaller, we never really did the work in computing the amount of gas each tx in a batch costs given an average batch size. This value is mutable and impacts end user gas prices, so we probably want to launch mainnet with the optimal value to pass savings on to users

Reply:
OK. Is this good for now? If it's mutable then I think it's fine to start with this then change later.

Reply:
Its good for now, we should be sure to measure on a live network and calculate what the mainnet value should be